open_schools_platform/organization_management/teachers/serializers.py

```diff
@@ -2,30 +2,30 @@
 from rest_framework import serializers
 
 from open_schools_platform.organization_management.teachers.models import TeacherProfile, Teacher
-from open_schools_platform.photo_management.photos.serializers import PhotoSerializer
+from open_schools_platform.photo_management.photos.serializers import GetPhotoSerializer
 
 
-class TeacherProfileSerializer(serializers.ModelSerializer):
-    photo = PhotoSerializer()
+class GetTeacherProfileSerializer(serializers.ModelSerializer):
+    photo = GetPhotoSerializer()
 
     class Meta:
         model = TeacherProfile
         fields = ("name", "age", "id", "phone", "photo")
 
 
-class QueryTeacherBodySerializer(serializers.ModelSerializer):
+class CreateTeacherBodySerializer(serializers.ModelSerializer):
```

Comment:
CreateTeacherBodySerializer is used in CreateCircleInviteTeacherSerializer.
![image](https://github.com/LamArt/open-schools-platform/assets/33755295/390217c6-e574-438e-bc53-0ce6903680ca)

Reply:
It is convinient to combine serializers only with one verb. It is also safetier because when you change get serializer it can't seriously break the system. So if developer changes post serializer he/she would do it carefully due to its serializer's name.
I think we should divide it with verbs